src/src_V2/components/container/Sidebar.jsx

```diff
@@ -1,186 +1,192 @@
+                            component={NavLink}
+                        >
+                            <ListItemText primary={title} />
+                        </ListItem>
+                    ))}
+                </List>
+            </Collapse>
+        </>
+    );
 }
 
 export default function Sidebar({ pageConfigs, open, setOpen }) {
-  const [expanded, setExpanded] = useState({});
-  const openCloseHandler = useCallback(() => setOpen(!open), [open, setOpen]);
-
-  const setExpandedState = useCallback(
-    (id) => {
-      setExpanded({
-        ...expanded,
-        [id]: !expanded[id],
-      });
-    },
-    [expanded]
-  );
-
-  return (
-    <Box style={boxStyles}>
-      <Tooltip placement={'right'} TransitionComponent={Zoom} title={!open ? 'Expand' : 'Collapse'}>
-        <Fab onClick={openCloseHandler} color={'inherit'} sx={fabStyles}>
-          {open ? <ChevronLeft /> : <ChevronRight />}
-        </Fab>
-      </Tooltip>
-      <Drawer variant="permanent" anchor="left" open={open}>
-        <List style={listStyles(open)}>
-          {pageConfigs.map((page) =>
-            page.path ? (
-              <MenuItem page={page} key={page.title} />
-            ) : (
-              <ExpandableMenuItem
-                open={open}
-                page={page}
-                key={page.title}
-                expanded={expanded}
-                setExpandedState={setExpandedState}
-              />
-            )
-          )}
-        </List>
-      </Drawer>
-    </Box>
-  );
+    const [expanded, setExpanded] = useState({});
+    const openCloseHandler = useCallback(() => setOpen(!open), [open, setOpen]);
+
+    const setExpandedState = useCallback(
+        (id) => {
+            setExpanded({
+                ...expanded,
+                [id]: !expanded[id],
+            });
+        },
+        [expanded]
+    );
+
+    return (
+        <Box style={boxStyles}>
+            <Tooltip placement={'right'} TransitionComponent={Zoom} title={!open ? 'Expand' : 'Collapse'}>
+                <Fab onClick={openCloseHandler} color={'inherit'} sx={fabStyles}>
+                    {open ? <ChevronLeft /> : <ChevronRight />}
+                </Fab>
+            </Tooltip>
+            <Drawer variant="permanent" anchor="left" open={open}>
+                <List style={listStyles(open)}>
+                    {pageConfigs.map((page) =>
+                        page.path ? (
+                            <MenuItem page={page} key={page.title} />
+                        ) : (
+                            <ExpandableMenuItem
+                                open={open}
+                                page={page}
+                                key={page.title}
+                                expanded={expanded}
+                                setExpandedState={setExpandedState}
+                            />
+                        )
+                    )}
+                </List>
+            </Drawer>
+        </Box>
+    );
```

Comment:
Is this eslint issue or some your editor config the indent is set to 4 and it's couses alot chnages, am i right?

Reply:
It's eslint config, the indent is set to 4. But currently, it will only throw warnings.

Reply:
@TigranYengiba04 jan, it seems pre-commit hook didn't work for you. Please follow these steps to make it work:

- After installing npm packages run `prepare` script for husky hooks to run properly
- Install lint-staged globally (npm i -g lint-staged)
- Editor support of Eslint and Prettier will turn the development process easier. Create a shortcut in your editor to run these tools on save. 

In Webstorm:
[run **Prettier** on save](https://prettier.io/docs/en/webstorm.html#using-prettier-in-webstorm)
[run **eslint --fix** on save](https://www.jetbrains.com/help/webstorm/eslint.html#ws_eslint_configure_run_eslint_on_save)

Reply:
done